src/vulkan/vulkan_renderer.cpp

```diff
@@ -1,8 +1,9 @@
 #include "vulkan_renderer.h"
 
-#include "vulkan_util.h"
```

Comment:
Went ahead and fixed this, but I think the pattern for includes we're going for is:

```cpp
#include "my_header.h"

#include <project_include_level_includes.h>

#include "project_same_dir_includes.h"

#include <third_party_lib_includes.h>

#include <std_libraries>
```